### PR TITLE
fix: change placeholder to input on time picker

### PR DESCRIPTION
**Pull request by stefanoScalzo**
#### Please provide a link to the associated issue.
fixes: #2851 
#### Please provide a brief summary of this pull request.
This pr makes the placeholder as not being hidden anymore. It is now an Input, the default is set to the type of time picker that is being used
#### Please check whether the PR fulfills the following requirements

- [x] the commit message follows the guideline:
https://github.com/SAP/fundamental-ngx/blob/master/CONTRIBUTING.md
- [x] tests for the changes that have been done
- [x] all items on the PR Review Checklist are addressed :
https://github.com/SAP/fundamental-ngx/wiki/PR-Review-Checklist

Documentation checklist:
- [x] update `README.md`
- [x] [Breaking Changes Wiki](https://github.com/SAP/fundamental-ngx/wiki/Breaking-Changes)
- [x] Documentation Examples
- [x] Stackblitz works for all examples

### CODE REVIEW

**Comment by InnaAtanasova** [comment]
I am confused. I selected Bulgarian and this is the format:
Screen Shot 2020-07-16 at 4 28 16 PM

Then I deleted the value and the Placeholder was:
Screen Shot 2020-07-16 at 4 28 26 PM

I switched to French, no changes for the Placeholder:
Screen Shot 2020-07-16 at 4 28 34 PM

Is it a Netlify bug or I am missing something?

File: libs/core/src/lib/time-picker/time-picker.component.ts [hunk 1]
```diff
@@ -123,17 +129,13 @@ export class TimePickerComponent implements ControlValueAccessor, OnInit {
...
-
     /** @hidden */
     onChange: Function = () => {};
     /** @hidden */
     onTouched: Function = () => {};
 
     /** @hidden */
     ngOnInit(): void {
-        this.placeholder = this.getPlaceholder();
     }
```
Comment (JKMarkowski) [inline libs/core/src/lib/time-picker/time-picker.component.ts:139]: LEt's remove ngOnInit + import + interface

Comment (stefanoScalzo) [reply libs/core/src/lib/time-picker/time-picker.component.ts]: fixed

File: apps/docs/src/app/core/component-docs/datetime-picker/examples/datetime-picker-complex-i18n-example/datetime-picker-complex-i18n-example.component.html [hunk 2]
```diff
@@ -31,4 +31,4 @@
         </fd-form-message>
     </div>
 </fieldset>
-<fd-datetime-picker [format]="actualFormat" [locale]="actualLocale" [(ngModel)]="date"></fd-datetime-picker>
+<fd-datetime-picker [placeholder]="placeholder" [meridian]="false"  [displaySeconds]=false [format]="actualFormat" [locale]="actualLocale" [(ngModel)]="date"></fd-datetime-picker>
```
Comment (salarenko) [inline apps/docs/src/app/core/component-docs/datetime-picker/examples/datetime-picker-complex-i18n-example/datetime-picker-complex-i18n-example.component.html:34]: Invalid code `[displaySeconds]=false`

File: apps/docs/src/app/core/component-docs/datetime-picker/examples/datetime-picker-complex-i18n-example/datetime-picker-complex-i18n-example.component.ts [hunk 3]
```diff
@@ -63,11 +63,22 @@ export class DatetimePickerComplexI18nExampleComponent {
...
     actualMomentJsLang: string = '';
 
+    placeholder: string = 'mm/dd/yyyy, hh:mm am'
+
     public date: FdDatetime = FdDatetime.getToday();
 
     public refresh(): void {
         this.datetimePickerComponent.locale = this.actualLocale;
         this.datetimePickerComponent.format = this.actualFormat;
+        console.log(this.actualLocale);
```
Comment (salarenko) [inline apps/docs/src/app/core/component-docs/datetime-picker/examples/datetime-picker-complex-i18n-example/datetime-picker-complex-i18n-example.component.ts:73]: console.log(this.actualLocale);

File: apps/docs/src/app/core/component-docs/datetime-picker/examples/datetime-picker-complex-i18n-example/datetime-picker-complex-i18n-example.component.ts [hunk 4]
```diff
@@ -63,11 +63,22 @@ export class DatetimePickerComplexI18nExampleComponent {
...
+        switch (this.actualLocale) {
```
Comment (salarenko) [inline apps/docs/src/app/core/component-docs/datetime-picker/examples/datetime-picker-complex-i18n-example/datetime-picker-complex-i18n-example.component.ts:74]: Formatting and small suggestion 😋
```
placeholders = new Map([
  ['en-gb', 'mm/dd/yyyy, hh:mm'],
  ['fr', 'dd/mm/yyyy  hh:mm']
]);

this.placeholder = this.placeholders.get(this.actualLocale);
```

**Pull request merged**